docs/setup.asciidoc

```diff
@@ -64,6 +64,17 @@ With this you enable every agent component including ASP.NET Core tracing, monit
 
 In case you only reference the `Elastic.Apm.AspNetCore` package, you won't find the `UseAllElasticApm`. Instead you need to use the `UseElasticApm()` method from the `Elastic.Apm.AspNetCore` namespace. This method turns on ASP.NET Core tracing, and gives you the opportunity to manually turn on other components. By default it will only trace ASP.NET Core requests - No HTTP request tracing, database call tracing or any other tracing component will be turned on.
 
+In case you would like to turn on specific tracing components you can pass those to the `UseElasticApm` method.
+
+For example:
+
+[source,csharp]
+----
+app.UseElasticApm(Configuration,
+	new HttpDiagnosticsSubscriber(),  /* Enable tracing of outgoing HTTP requests */,
+	new EfCoreDiagnosticsSubscriber() /* Enable tracing of database calls through EF Core*/)
```

Comment:
Is it standard to put the closing parenthesis after the inline comment? If so, LGTM.

Reply:
Looking at how this is rendered with a scrollbar in our docs, I'd be tempted to move that parenthesis to before the comment:

<img width="770" alt="Screen Shot 2020-01-14 at 11 13 31 AM" src="https://user-images.githubusercontent.com/5618806/72374357-f6c4cc00-36be-11ea-980b-be2c5dc1a762.png">

Reply:
> Is it standard to put the closing parenthesis after the inline comment?

I pasted it to my editor and auto-formatted, but as it seems the C# auto-formatter with our configs does not care about it, it leaves both versions as it is. I took your suggestion, let's roll with that.